### ISSUE TITLE
TypeError: 'undefined' is not a function (evaluating 'session.load()')

### ISSUE THREAD

**Issue by janvanhelvoort** [SAPikachu/icomoon-build#7]
Running the next command:

` $ icomoon-build -p icomoon/icomoon-build.json --scss scss/output.scss --fonts fonts/`

resulting in the following error:
```
Error: Command failed: c:\Users\jan\AppData\Roaming\npm\node_modules\icomoon-bui
ld\node_modules\phantomjs\lib\phantom\phantomjs.exe --load-images=false --ssl-pr
otocol=any c:\Users\jan\AppData\Roaming\npm\node_modules\icomoon-build\lib\icomo
on.phantom.js icomoon/icomoon-build.json
onNavigationRequested: http://icomoon.io/app/
type: Other
willNavigate: true
main: true
 --
onLoadStarted: about:blank
 --
onNavigationRequested: https://icomoon.io/app/
type: Other
willNavigate: true
main: true
 --
onInitialized: https://icomoon.io/app/
 --
CONSOLE: [PhantomXHR] Match added [All REST verbs] : .*icomoon-lib\/presets\.jso
n' (from line #undefined in "undefined")
CONSOLE: [PhantomXHR] Match added [All REST verbs] : .*\.json' (from line #undef
ined in "undefined")
CONSOLE: [PhantomXHR] Match added [All REST verbs] : .*authstat.*' (from line #u
ndefined in "undefined")
CONSOLE: [PhantomXHR] Match added [All REST verbs] : .*getsessiontime' (from lin
e #undefined in "undefined")
CONSOLE: [PhantomXHR] Match added [All REST verbs] : .*' (from line #undefined i
n "undefined")
onNavigationRequested: https://icomoon.io/app/#/select
type: Other
willNavigate: true
main: true
 --
onLoadFinished: https://icomoon.io/app/#/select
Status: success
 --
ERROR: TypeError: 'undefined' is not a function (evaluating 'session.load()')
TRACE:
 -> phantomjs://webpage.evaluate(): 19
 -> phantomjs://webpage.evaluate(): 39
 -> phantomjs://webpage.evaluate(): 39
    at ChildProcess.exithandler (child_process.js:751:12)
    at ChildProcess.emit (events.js:110:17)
    at maybeClose (child_process.js:1015:16)
    at Process.ChildProcess._handle.onexit (child_process.js:1087:5)
```

If you require any additional information to resolve this please let me know, would help gladly

**Comment by SAPikachu** [comment]
Can you upload your project file (`icomoon/icomoon-build.json`) to somewhere and post the link here (or send to i@sapika.ch)? I will have a look on it this weekend.

**Comment by SAPikachu** [comment]
Fixed in 0.2.4. @janvanhelvoort Can you try the new version?

**Comment by janvanhelvoort** [comment]
Yes, it works :smiley: Thank you :+1:
